src/locale/translations/en.json

```diff
@@ -152,5 +152,8 @@
   },
   "Partners": {
     "Label": "Developed in partnership with"
+  },
+  "BottomSheet": {
+    "Collapse": "Collapse"
```

Comment:
Can we change this to "Close" since my recommended label for the closed state is, "Open". https://github.com/CovidShield/mobile/issues/37